generate_manifest.py

```diff
@@ -62,15 +62,17 @@ def required_to_deploy_this_project(project, specific_projects):
                 if specific_versions[project] is None:
                     manifest['Projects'][project] = project_detail = None
                     continue
-                deployment = octo.get_deploy_for_version(project_id, specific_versions[project])
+                release = octo.get_release_for_version(project_id, specific_versions[project])
                 project_detail['Version'] = specific_versions[project]
-                project_detail['Packages'] = octo.get_specific_packages(deployment)
+                project_detail['Packages'] = octo.get_specific_packages(release)
             elif env != "local":
-                deployment = octo.get_deploy_for_env(project_id, environments[env])
-                project_detail['Version'] = deployment['Version']
-                project_detail['Packages'] = octo.get_specific_packages(deployment)
+                release = octo.get_release_for_env(project_id, environments[env])
+                project_detail['Version'] = release['Version']
+                packages = octo.get_specific_packages(release, environments[env])

```

Comment:
I cant remember the specific semantics around Octopus but we need to make sure that we are not skipping the manual verification steps by deduplicating the steps on none local environments. 

Specifically the double database step was originally used to get confidence in the roll forward before the rollforward was executed.

Reply:
So in `remote_deploy.py` it has:
```
to_deploy = octo.get_release_for_version(proj_id, version)
...
deploy = octo.deploy_release(to_deploy['Id'], env_id)
```
and this should deploy the selected Release to the Environment, following all the steps in the process. It does do some logging of what packages are being deployed, but I think it lets Octopus handle it and it will therefore take whatever is configured on the Octopus Server.

Reply:
This covers the double db deploy step okay?

Reply:
Ahhh okay, now i remember how this works, yeah your right it will just kick octopus to do the right thing.